csrc/mmdeploy/backend_ops/tensorrt/bicubic_interpolate/trt_bicubic_interpolate.cpp

```diff
@@ -42,13 +42,12 @@ nvinfer1::DimsExprs TRTBicubicInterpolate::getOutputDimensions(
   ret.nbDims = 4;
   ret.d[0] = inputs[0].d[0];
   ret.d[1] = inputs[0].d[1];
-  auto height = exprBuilder.constant(mScaleFactor[0]);
-  auto width = exprBuilder.constant(mScaleFactor[1]);
-  auto d2 = exprBuilder.operation(DimensionOperation::kPROD, *inputs[0].d[2], *height);
-  auto d3 = exprBuilder.operation(DimensionOperation::kPROD, *inputs[0].d[3], *width);
-  ret.d[2] = d2;
-  ret.d[3] = d3;
-
+  auto height = (double)inputs[0].d[2]->getConstantValue();
+  auto width = (double)inputs[0].d[3]->getConstantValue();
+  auto outHeight = (int)height * mScaleFactor[0];
+  auto outWidth = (int)width * mScaleFactor[1];
+  ret.d[2] = exprBuilder.constant(outHeight);
```

Comment:
Use exprBuilder.constant will fix the output shape. I don't think this is expected.

Reply:
https://github.com/NVIDIA/TensorRT/issues/347
I can't seem to make it expectable.
Probably there is no workaround.
What do you think about it?

Reply:
Nope, unless you want to use the shape generated by IResizeLayer.